device/mellanox/x86_64-mlnx_msn2700-r0/platform.json

```diff
@@ -0,0 +1,484 @@
+{
+    "chassis": {
+        "name": "Mellanox-SN2700",
+        "model": "FIXME",
+        "serial": "FIXME",
```

Comment:
Serial number is device-specific, not platform-specific, and thus will vary. Please remove.

Reply:
fixed